### ISSUE TITLE
Negative timeout in work-parameters

### ISSUE THREAD

**Issue by blueyed** [sixty-north/cosmic-ray#233]
I am getting these `work-parameters`:

```
   "work-parameters" : {
      "1" : {
         "timeout" : -69379.47031,
         "test-args" : [
            "foo/tests"
         ],
         "test-runner" : "pytest"
      }
```

I've used `cosmic-ray init --baseline=10 …` from the README.

When running it with pdb `t.elapsed.total_seconds()` in `handle_init` was `-6620.261833` - `t.elapsed` being `datetime.timedelta(-1, 79785, 202052)`.

```
((Pdb++)) self._start
datetime.datetime(2017, 9, 8, 14, 26, 9, 70743)
((Pdb++)) datetime.datetime.now()
datetime.datetime(2017, 9, 8, 12, 37, 43, 476677)
```

Looks like one is using UTC and the other local time?!

I will provide a PR later to fix this eventually - might just use `utcnow()` there.

**Comment by abingham** [comment]
That is truly weird! Does this happen consistently? And does switching to `utcnow()` fix things?

**Comment by blueyed** [comment]
I assume it will fix it, and that `datetime` somehow gets mutated during tests - I am just trying it a bit on the side just now.

**Comment by abingham** [comment]
You're probably right about the baseline testing screwing with `datetime`. And this really points to deeper issue that we should fix: #182. In a nutshell, test runs against mutants are done in a subprocess. This is specifically to prevent the kind of corruption you're seeing.

However, we don't run the baseline test run in a separate process. I naively assumed that un-mutated test suites would be benign. So I think what we'll probably need to do is run the baseline in a worker process, and there's a bit of infrastructure work to do for that.

**Comment by abingham** [comment]
With that said, if `utcnow()` gets you past your current roadblock, I'm happy to merge that in.

**Comment by abingham** [comment]
I think this should be fixed as of 54e2424dcb74f01d1b0b040f873d538386107ef0 (merged into master). Could you see if things work for you now without the `utcnow()` change?

**Comment by blueyed** [comment]
Looks like a good fix.
Will have to find some time for this (not in the near future).
Let's assume this is fixed.
